src/views/TrialDescriptionPage/TrialDescriptionPage.jsx

```diff
@@ -1,7 +1,7 @@
 import queryString from 'query-string';
 import React, { useEffect, useReducer } from 'react';
 import { Helmet } from 'react-helmet';
-import { useDispatch, useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
```

Comment:
Is this still being used anywhere in this file?

Reply:
Yep, otherwise the linters would complain. It's still being used by as rdx_dispatch to update the form criteria. 	`rdx_dispatch(updateFormSearchCriteria(res.searchCriteria));`

Whether or not that's still required / should be addressed by this ticket is another story...

Reply:
Ah, found its use in the file. Couldn't when I checked earlier. 👍